docs/whats-new/dotnet-7-docs.md

```diff
@@ -0,0 +1,295 @@
+- [MSBuild reference for .NET SDK projects](../core/project-sdk/msbuild-props.md)
+  - Add three new properties
+  - Add docs for project properties that control PDBs and XML publishing
+  - Update the documentation for PublishRelease and PackRelease
+  - Add DisableTransitiveProjectReferences
+  - Cleanup trimming options and document TrimMode=full,partial
+  - Document new PublishRelease and PackRelease properties
+- [How to enable tab completion for the .NET CLI](../core/tools/enable-tab-autocomplete.md) - Merge .NET 7 release branch into main
+- [Manage package dependencies in .NET applications](../core/tools/dependencies.md) - Add three new properties
+- [.NET default templates for dotnet new](../core/tools/dotnet-new-sdk-templates.md) - Added the two new Blazor empty templates
+- [.NET SDK and .NET CLI telemetry](../core/tools/telemetry.md)
+  - Add the PublishProtocol property to the list of telemetry data disclosures
+  - document SDK telemetry from prior and upcoming versions
+- [dotnet new install](../core/tools/dotnet-new-install.md) - New syntax for `dotnet new`
+- [dotnet new list](../core/tools/dotnet-new-list.md) - New syntax for `dotnet new`
+- [dotnet new search](../core/tools/dotnet-new-search.md) - New syntax for `dotnet new`
+- [dotnet new uninstall](../core/tools/dotnet-new-uninstall.md) - New syntax for `dotnet new`
+- [dotnet new update](../core/tools/dotnet-new-update.md) - New syntax for `dotnet new`
+- [dotnet new &lt;TEMPLATE&gt;](../core/tools/dotnet-new.md) - New syntax for `dotnet new`
+- [dotnet restore](../core/tools/dotnet-restore.md) - Document .NET 7 SDK changes to NuGet signed package verification
+- [dotnet publish](../core/tools/dotnet-publish.md) - Document Publish UseCurrentRuntime
+- [dotnet build](../core/tools/dotnet-build.md) - Document UseCurrentRuntime
+- [dotnet watch](../core/tools/dotnet-watch.md) - Note that --non-interactive is .NET 7
+- [dotnet nuget sign](../core/tools/dotnet-nuget-sign.md) - Document .NET 7 SDK changes to NuGet signed package verification
+- [dotnet nuget trust](../core/tools/dotnet-nuget-trust.md) - Document .NET 7 SDK changes to NuGet signed package verification
+- [dotnet nuget verify](../core/tools/dotnet-nuget-verify.md) - Document .NET 7 SDK changes to NuGet signed package verification
+- [.NET SDK error list](../core/tools/sdk-errors/index.md)
+  - Add new errors/warnings to NETSDK error index
+  - Document selected NETSDK error messages
+  - List of NETSDK error messages
+- [NETSDK1100: Set the EnableWindowsTargeting property to true](../core/tools/sdk-errors/netsdk1100.md)
+  - Add three new properties
+  - Document selected NETSDK error messages
+- [dotnet-coverage code coverage utility](../core/additional-tools/dotnet-coverage.md) - Adding new options and example scenarios
+- [.NET additional tools overview](../core/additional-tools/index.md) - Add dotnet-monitor .NET tool documentation
```

Comment:
Is there a reasonable overview page for the SDK updates?

Reply:
@tdykstra

Reply:
Both the _introduction.md_ and _sdk.md_ files were updated by Rich Lander, it wouldn't hurt to include them here. See https://github.com/dotnet/docs/commit/22aedbeaa493b18631686756174c30945993db3d

Reply:
> Is there a reasonable overview page for the SDK updates?

I don't believe so, but [dotnet new &lt;TEMPLATE&gt;](../core/tools/dotnet-new.md#description) would work as an overview for all of the dotnet new articles.